frontend/tsconfig.json

```diff
@@ -25,12 +25,13 @@
     "paths": {
       "react": ["./node_modules/@types/react"]
     },
-    "incremental": true
+    "incremental": true,
```

Comment:
Can you remove the comma here? My editor will automatically remove trailing commas from JSON files when I save the file.